doc/topics/cloud/gce.rst

```diff
@@ -235,6 +237,13 @@ Additionally, the subnetwork your instance is created under is associated with t
 
 .. versionadded:: 2017.7.0
 
+labels
+----
+
+This setting allows you to set labels on your GCE instances. It
+should be a dictionary and must be parse-able by the python
+ast.literal_eval() function to convert it to a python dictionary.
+
```

Comment:
can you add a versionadded tag here: `.. versionadded:: 3006`

Reply:
done!